administrator/components/com_templates/src/Model/TemplateModel.php

```diff
@@ -1810,11 +1810,23 @@ public function getMediaFiles()
         $path               = Path::clean(JPATH_ROOT . '/media/templates/' . ($template->client_id === 0 ? 'site' : 'administrator') . '/' . $template->element . '/');
         $this->mediaElement = $path;
 
+        if (!is_dir($path)) {
+            // Just in case a template does not include any media
+            mkdir($path, 0777, true);
+        }
+
+        $result = $this->getDirectoryTree($path);
+
         if (!is_writable($path)) {
             $app->enqueueMessage(Text::_('COM_TEMPLATES_DIRECTORY_NOT_WRITABLE'), 'error');
         }
+        else if (empty($result)) {
+            // Add some default folders
+            foreach(['css', 'images', 'js', 'sccs'] as $folder)
+            {
```

Comment:
```suggestion
        } elseif (empty($result)) {
            // Add some default folders
            foreach(['css', 'images', 'js', 'sccs'] as $folder) {
```
Fix PHP code style check failing in drone.